docs/_quarto.yml

```diff
@@ -188,13 +185,99 @@ quartodoc:
     - title: Expression API
       desc: "APIs for manipulating table, column and scalar expressions"
       package: ibis.expr.types
+      contents:
+        - kind: page
+          path: expression-tables
+          package: ibis.expr.types.relations
+          summary:
+            name: Table
```

Comment:
Dropped the "expression" suffix from all these sections. This was named "Table expressions" now just "Table". Similar for the other secitons.